sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -16,7 +16,7 @@
               v-model="legalName"
               data-cy="legalName"
               @keyup.enter="searchInstitutions"
-              hide-details
+              hide-details="auto"
```

Comment:
We centralized the validations in the `v-input`, do we need auto `hide-details="auto"` here?